src/pages/unsupported-browser.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import UnsupportedBrowserPage from "../page-unsupported-browser/UnsupportedBrowserPage";
```

Comment:
Would you mind also updating the copy for this page now that we will be able to support IE11? Something like, "We’re sorry, legacy browsers such as versions of Internet Explorer prior to IE11 are not..."

Reply:
Oh, this page doesn't work right now -- I took out the logic to detect old browsers. (I could have left it in, but there's no guarantee that it would have worked, so I didn't want to give a false sense of security.) I'll create an issue so that we can decide whether to add in that functionality (and what that will look like) or just remove it.